frontend-react/docs/patches.md

```diff
@@ -0,0 +1,26 @@
+# Patches
```

Comment:
I think this piece of information should be in our `README`?

Reply:
As a link, for sure, but I don't think it should clutter the main readme.